spec/unit/api_checker_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+require 'shopify_theme'
+require 'shopify_theme/api_checker'
+
+module ShopifyTheme
+  describe "APIChecker" do
+    include WebMock::API
+    attr_reader :checker
```

Comment:
:police_car: Could you add a blank line before and after this line, please? Whitespace FTW.